app/scripts/keyring-controller.js

```diff
@@ -173,10 +173,15 @@ module.exports = class KeyringController extends EventEmitter {
     })
   }
 
-  placeSeedWords () {
+  placeSeedWords (cb) {
     const firstKeyring = this.keyrings[0]
     const seedWords = firstKeyring.serialize().mnemonic
     this.configManager.setSeedWords(seedWords)
+
+    if (cb && typeof cb === 'function') {
```

Comment:
This method is sometimes called synchronously. Look at the code context to see why.